server/src/main/resources/config/windows/nodeConfig.json

```diff
@@ -29,6 +29,7 @@
         "port": 5555,
         "host": "ip",
         "register": true,
-        "hubPort": 4444
+        "hubPort": 4444,
+        "nodeRecycleThreadWaitTimeout": 300
```

Comment:
Well, the point is Selenium provides a path for hub and node configuration and a registry to keep track of it.. There's nothing about the selenium config implementation that is strictly selenium-core specific.

One key advantage to this approach is the hub can set value X and each node can override it with value Y.. The existing `SeLionConfig.json` approach does not support this -- We'd have to change the node boot code to include the override with the registration request --- which at that point is going to put the new value in the exact same config (selenium) source.